tests/sentry/api/endpoints/test_group_details.py

```diff
@@ -689,6 +692,15 @@ def test_delete(self):
         # Now we killed everything with fire
         assert not Group.objects.filter(id=group.id).exists()
         assert not GroupHash.objects.filter(group_id=group.id).exists()
+        with self.tasks():
```

Comment:
```suggestion
        with self.tasks(), outbox_runner():
```

This produces a different error but I think you might just need to tweak the `get` query below:

```
E   sentry.models.auditlogentry.AuditLogEntry.MultipleObjectsReturned: get() returned more than one AuditLogEntry -- it returned 2!
```